src/masternodes/mn_checks.cpp

```diff
@@ -3705,6 +3756,106 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
         return !res ? res : mnview.StoreAuctionBid({obj.vaultId, obj.index}, {obj.from, obj.amount});
     }
 
+    Res operator()(const CCreatePropMessage& obj) const
+    {
+        auto res = IsOnChainGovernanceEnabled();
+        if (!res) {
+            return res;
+        }
+
+        switch (obj.type) {
+            case CPropType::CommunityFundProposal:
+                if (!HasAuth(obj.address))
+                    return Res::Err("tx must have at least one input from proposal account");
+
+                if (obj.nCycles < 1 || obj.nCycles > MAX_CYCLES)
```

Comment:
Seems like a duplicate check of Line 3812. 

https://github.com/DeFiCh/ain/blob/4c7ec5c898c482472d25a9e06c5aad83d3eeaea9/src/masternodes/mn_checks.cpp#L3812

Reply:
It is removed in #1543 which is not yet merged into this PR.